### DISCUSSION TITLE
PositionAware not found

### DISCUSSION THREAD

**Discussion by hitheshsai** [halaxa/json-machine#49] [Q&A] [answered]
Hi I am facing this error when trying to use the decoder.

`( ! ) Fatal error: Interface 'JsonMachine\PositionAware' not found in /var/www/esm/htdocs/sportlomo/vendor/halaxa/json-machine/src/JsonMachine.php on line 8`

**Comment by halaxa** [comment, marked as answer]
Hi, can you show the code, which causes this? It seems to be a problem with the package installation. Did you clone the repository or use Composer?

**Reply by hitheshsai** [reply]
Hey. I tried cloning the repository as I do development on one pc and server runs on other machine via vagrant. But now it works fine as I set it up using composer from a different pc. Thanks
